### PR TITLE
add bitlish.com to cryptocurrencies

### PR DESCRIPTION
**Pull request by margarita-shishkina**

### CODE REVIEW

**Comment by mxxcon** [comment]
Please add this entry into its correct alphabetical spot.
Are there any docs that document what type of tfa bitlish supports?

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -221,3 +221,11 @@ websites:
       tfa: Yes
       software: Yes
       doc: https://twitter.com/jstcoin/status/445900548743897088
+    - name: BITLISH
+      img: bitlish.png
+      url: https://bitlish.com
+      img: btc-e.png
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]: This img tag points to `btc-e.png` and not `bitlish.png`.
bitlish.png is also larger than we allow. Here's a compressed version of the image:
bitlish compressed

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -31,6 +31,15 @@ websites:
...
       doc: https://bitgo.zendesk.com/hc/en-us/sections/201009049-Two-Factor-Authentication-Help-Page
 
+    - name: Bitlish
+      url: https://bitlish.com
+      img: bitlish.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      phone: Yes
+      doc: https://www.authy.com
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml:41]: Please change the doc link to a documentation page by Bitlish.

**Comment by Carlgo11** [comment]
@margarita-shishkina are you going to update the PR?
Please respond within a couple of days or I'll close the PR due to inactivity.

**Comment by jamcat22** [comment]
Closing and adding to backlog due to inactivity.

**Pull request closed without merging**